### DISCUSSION TITLE
请问overleaf编译出来的文档是否字体有区别

### DISCUSSION THREAD

**Discussion by Coding-QQ** [tuna/thuthesis#978] [Q&A] [answered]
我设置了fontset=windows，通过本地的编译器编译出来后一切正常。然而，发现overleaf编译出来的文档中，字体与正常的宋体存在一定的差别，如下图所示：
![image](https://github.com/user-attachments/assets/6d71a9ff-00d6-46a2-8642-60a2b6a6c2bf)
但正常情况下，应该是下面的样子：
![image](https://github.com/user-attachments/assets/1e3141f1-fa35-4ff8-b653-8933d9ae1665)

**Comment by Harry-Chen** [comment, marked as answer]
Overleaf 没有 Windows 字体，理论上无法使用该 `fontset=windows`。具体使用的是什么字体，可以在日志中搜索 `Detected CJK font:` 字样/

图中的字体看起来是 Fandol。

**Reply by Coding-QQ** [reply]
哦哦明白了，也就是说这个模板最后要编译的话，只能在本地的编译器中编译嘛？

**Reply by Harry-Chen** [reply]
TeXPage 等在线服务提供了相关字体，您也可以自行上传字体到 Overleaf，文档中有说明。

**Reply by Coding-QQ** [reply]
感谢！！
我在"C:\Windows\Fonts"文件夹下，找到了"simsun.ttc", "simhei.ttf", "simkai.ttf", "simfang.ttf"共四个字体文件（分别对应宋体、黑体、楷体、仿宋），并将它们上传到了overleaf项目的根目录下。此时再设置"fontset=windows"就可以正常显示字体了。

**Reply by lyfdy456** [reply]
兄弟你是神！！！今天一直找不到原因，现在按照你的方法成了

**Reply by ChrisZhao22** [reply]
无敌了！按照这个方法在mac上也能编译出来windows上的中文字体，悬着的心终于放下来了！

**Comment by Coding-QQ** [comment]
补充一下，如果我在文章开头使用：\documentclass[fontset=windows]{thuthesis}，编译后会报错，并显示：Class thuthesis Error: Cannot find "SimSun" font.
如果我在thusetup.tex中输入命令：\thusetup{fontset=windows,}，则不会有任何区别，也不会报错
